src/components/admin/Forms.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState, FormEvent } from "react";
+import "../../App.css";
+import FormTitle from "./FormTitle.tsx";
+import FormAdd from "./FormAdd.tsx";
+import { createDefaultFormObj } from "../../util/utils.ts";
+import { FormPageData } from "../../types/type.ts";
+import FormItem from "./FormItem.tsx";
+import cloneDeep from "lodash.clonedeep";
+
+function Forms() {
+  const copiedObj = cloneDeep(createDefaultFormObj());
+  const [optionGroup, setOptionGroup] = useState<FormPageData>(copiedObj);
+
+  const onClickSubmit = async (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    const result = await fetch("/api", {
+      method: "POST",
+      body: JSON.stringify(optionGroup),
+    });
+    const response = await result.json();
+    console.log(response);
+  };
+
+  const onClickAdd = () => {
+    const copiedForms = cloneDeep(createDefaultFormObj().forms[0]);
+    setOptionGroup((prev) => ({
+      ...cloneDeep(prev),
+      forms: [...prev.forms, copiedForms],
+    }));
```

Comment:
P3;

요렇게 하면 한번만 cloneDeep 할 수 있을 것 같아요.
위에 남긴 코멘트 처럼 cloneDeep이 꼭 필요한 상황에서만 사용하는 것을 권장드려요! 

```suggestion
const copiedForms = createDefaultFormObj().forms[0];
  setOptionGroup((prev) => {
    const clonedPrev = cloneDeep(prev);
    clonedPrev.forms.push(copiedForms);
    return clonedPrev;
  });
```